src/test.rs

```diff
@@ -558,3 +559,68 @@ fn accept_one_byte_at_a_time() {
 
     t.join().unwrap();
 }
+
+#[test]
+fn split_cert_key() {
+    let listener = TcpListener::bind("127.0.0.1:0").unwrap();
+    let addr = listener.local_addr().unwrap();
+    let t = thread::spawn(move || {
+        let cert = include_bytes!("../test/cert.der");
+        let mut store = Memory::new().unwrap();
+        store.add_encoded_certificate(cert).unwrap();
+        let store = store.into_store();
+
+        let stream = TcpStream::connect(&addr).unwrap();
+        let creds = SchannelCred::builder()
+                                 .acquire(Direction::Outbound).unwrap();
+        let mut stream = tls_stream::Builder::new()
+            .domain("foobar.com")
+            .cert_store(store)
+            .connect(creds, stream)
+            .unwrap();
+        stream.write_all(&[1, 2, 3, 4]).unwrap();
+        stream.flush().unwrap();
+        assert_eq!(stream.read(&mut [0; 1024]).unwrap(), 4);
+        stream.shutdown().unwrap();
+    });
+
+    let cert = include_bytes!("../test/cert.der");
+    let cert = CertContext::new(cert).unwrap();
+
+    let mut options = AcquireOptions::new();
+    options.container("schannel-test");
```

Comment:
If I understand CryptAcquireContext correctly, if you don't use `CRYPT_VERIFYCONTEXT`, the key is stored permanently on disk, is that right?